pkg/porter/credentials.go

```diff
@@ -139,15 +140,10 @@ func (p *Porter) GenerateCredentials(opts CredentialOptions) error {
 
 // Validate validates the args provided Porter's credential show command
 func (o *CredentialShowOptions) Validate(args []string) error {
-	switch len(args) {
-	case 0:
-		return errors.Errorf("no credential name was specified")
-	case 1:
-		o.Name = strings.ToLower(args[0])
-	default:
-		return errors.Errorf("only one positional argument may be specified, the credential name, but multiple were received: %s", args)
+	if err := validateCredentialName(args); err != nil {
```

Comment:
I was gonna comment on this not being a receiver function but it's the notorious validate name that I was just saying to Vaughn we need a common function for across everything. I'm gonna fix that today after your PR goes through because it's buggin' me. 😀